recipe/gen_patch_json.py

```diff
@@ -1061,6 +1061,13 @@ def _gen_new_index(repodata, subdir):
                 i = record["depends"].index("py-tools-ds >=0.15.10")
                 record["depends"][i] = "py-tools-ds >=0.15.10,<=0.15.11"
 
+        # https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/issues/159
+        if record_name == "snowflake-sqlalchemy" and record["version"] == "1.3.1" and record["build_number"] == 0:
```

Comment:
Have a look at the section above. That one is a bit neater as it doesn't copy the list of dependencies but just replaces the offending entry.

Reply:
Done.